src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -220,6 +220,8 @@ public function set_quality( $quality = null ) {
 					}
 					break;
 				case 'image/avif':
+					$this->image->setOption( 'heic:speed', 7 );
```

Comment:
What about `setImageCompressionQuality`?

Reply:
> What about setImageCompressionQuality?

The speed variable has to do with how hard the compressor works to try to get to the quality which is a separate setting.

The avif case will fall thru to L:226 where the quality is set (not working for AVIF, but that is another ticket).

Reply:
I guess I can remove the fall thru since none of the other cases do that and its prone to future issues.

Reply:
done in 8224ffb3327afe87248aa64be667288a3363e49c